pkg/testing/http_mock.go

```diff
@@ -18,16 +18,21 @@ package testing
 
 import (
 	"bufio"
+	"fmt"
 	"io/ioutil"
 	"net"
 	"net/http"
 	"os"
-	"path/filepath"
 	"regexp"
-	"runtime"
+	"strings"
 	"time"
 )
 
+const (
+	OCTET_STREAM = "application/octet-stream"
+	JSON         = "application/json; charset=utf-8"
```

Comment:
>  find the names a bit too long then and I think they are telling enough.

For me, they are telling generic meaning. I am fine with a little longer name over to navigate over to  the definition to understand what a constant is doing. Let's get the feedback of others too. 
cc @gbraad @LalatenduMohanty @praveenkumar .